### ISSUE TITLE
Tables doesn't displaying in Heroku

### ISSUE THREAD

**Issue by IvanGodinez21** [ayonious/console-table-printer#383]
When I try to use it in Heroku (A PaaS that works using Ubuntu), tables doesn't display as should be

![image](https://user-images.githubusercontent.com/45635827/125169278-08314c00-e16f-11eb-95e2-0cdd7026be85.png)

**Comment by ayonious** [comment]
Yes I have noticed this before while running the tests on Heroku/circle-ci/travis. They use a different Terminal that what we use in local. 

The solution is I need to find a way to detect terminal type and use proper ascii color code for that terminal.

**Comment by ArjixWasTaken** [comment]
A lazy and easy fix would be to rely on an environment variable to choose what ascii color codes to use.

**Comment by ayonious** [comment]
@ArjixWasTaken sounds good

**Comment by ayonious** [comment]
@IvanGodinez21 is this not happening anymore?

**Comment by IvanGodinez21** [comment]
Oh, @ayonious I didn't see any update related to this issue, so I decided to close it, but if you want to re-open it, you can do it. Maybe this issue could be hard to fix, and not just in Heroku, there are many other platforms that may have this bug.

I leave it to your decision

**Comment by ayonious** [comment]
With this following feature no more special char would be printed which is lot better than having broken CI views.
https://console-table.netlify.app/docs/doc-no-ascii

I wont consider this completely fixed but at least the basic table view should be seen using this. Styling can be done later
